docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -0,0 +1,213 @@
+point to pages that handle these operations in your setup. In this example, the sample application runs at
+`http://localhost:3000`.
+
+Follow these steps to configure Ory Cloud to call the sample application for login and consent screens:
+
+1. Download the configuration of your Ory Cloud project and save it to a `yaml` file:
+
+   ```shell
+   ## List all available projects
+   ory list projects
+
+   ## Get config
+   ory get project <project-id> --format yaml > config.yaml
+   ```
+
+2. Adjust the configuration in `oauth2/urls/consent` and `oauth2/urls/login`:
+
+   ```yaml title="config.yaml"
+     oauth2:
+     # ...
+       urls:
+       // highlight-next-line
+         consent: http://localhost:3000/consent
+         error: /oauth2/fallbacks/error
+       // highlight-next-line
+         login: http://localhost:3000/login
+         post_logout_redirect: /oauth2/fallbacks/logout
+   ```
+
+3. Update the Ory Cloud project configuration using the file you worked with:
+
+   ```shell
+   ory update project <project-id> --file config.yaml
+   ```
+
+### Create a local web server acting as OAuth2 client
+
+Use the Ory Hydra CLI to create a sample web server that acts as the OAuth2 client. To successfully perform the Authorization Code
+Grant flow, the client ID and client secret must be registered in Ory Cloud.
+
+Run this command to create the client. Using flags, provide the client ID and secret of the client created in Ory Cloud:
+
+```bash
+hydra token user \
+    --client-id CLOUD_CLIENT_ID \
+    --client-secret CLOUD_CLIENT_SECRET \
+    --endpoint SDK_CONFIGURATION_URL \ # Found in the 'Connect' section of the Cloud Console.
+    --port 5555 \
+    --scope openid,offline
+```
+
+When this command runs successfully, a browser window opens automatically and displays a welcome page. If this doesn't happen, go
+to `http://127.0.0.1:5555/`.
+
+## Run the flow
+
+After completing the configuration steps, you can start the flow from the welcome page at `http://127.0.0.1:5555/`.
+
+:::tip
+
+The login and consent application at `http://localhost:3000/` shows live logs in its terminal session. Inspecting them can give
+you more insight into how this example works.
+
+:::
+
+To execute the flow.
+
+1. In a browser window, go to `http://127.0.0.1:5555/` and click **Authorize application**.
+2. On the login screen, provide the user details and log in to proceed.
+3. Choose the scopes you want to give the client access to and click **Allow access**. You must choose at least one scope.
+4. The next page shows the tokens the client got as a result of running the Authorization Code Grant flow.
+
+:::note
+
+You can get a closer look at the `access_token` by decoding it at [jwt.io](https://jwt.io/). In the decoded token, you can find
+information about the user on whose behalf the client acts and the URL of the issuer - your Ory Cloud project.
+
+:::
```

Comment:
Let's add a nice "that's it - you've now succeeded and your client now has access to ..." message to make sure the reader understood what just happend.